filebeat/harvester/reader/json.go

```diff
@@ -120,6 +120,9 @@ func (r *JSON) Next() (Message, error) {
 	if err != nil {
 		return message, err
 	}
-	message.Content, message.Fields = r.decodeJSON(message.Content)
+
+	var fields = common.MapStr{}
```

Comment:
prefer `var fields common.MapStr` to have a `nil` map by default.

The line as is creates a new map to discard it on next line again.